src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -596,7 +596,19 @@ def get_clouds(cli_ctx):
             # If profile isn't set, use latest
             setattr(c, 'profile', 'latest')
         if c.profile not in API_PROFILES:
-            raise CLIError('Profile {} does not exist or is not supported.'.format(c.profile))
+            if c.profile in (
+                "2017-03-09-profile",
+                "2018-03-01-hybrid",
+                "2019-03-01-hybrid",
+                "2020-09-01-hybrid",
+            ):
+                logger.error(
+                    "The azure stack profile '%s' has been deprecated and removed, using the 'latest' profile instead.\n"
+                    "To continue using Azure Stack please install the Azure CLI `2.66.*` (LTS) version.", c.profile
```

Comment:
Do we have any guidelines or docs for LTS versions? If so, could we put a document link here, maybe could help customers better understand what LTS versions are